spec/cordova/plugin/save.spec.js

```diff
@@ -54,15 +61,145 @@ describe('cordova/plugin/save', function () {
         });
     });
     describe('happy path', function () {
-        it('should remove all plugins from config.xml and re-add new ones based on those retrieved from fetch.json');
-        it('should only add top-level plugins to config.xml');
-        it('should write individual plugin specs to config.xml');
-        it('should write individual plugin variables to config.xml');
+        it('should remove all plugins from config.xml and re-add new ones based on those retrieved from fetch.json', function (done) {
```

Comment:
This test checks that the plugins get removed but not if the new ones get re-added (based on description). I think the description should be updated to just check that existing plugins are getting removed